src/ecs/commands/updateEnableExecuteCommandFlag.ts

```diff
@@ -58,7 +65,8 @@ export async function updateEnableExecuteCommandFlag(
         await node.ecs.updateService(node.service.clusterArn!, node.name, enable)
         result = 'Succeeded'
         node.parent.clearChildren()
-        commands.execute('aws.refreshAwsExplorerNode', node.parent)
+        node.parent.refresh()
+        window.showInformationMessage(updatingServiceMessage)
```

Comment:
We should add a backlog item to turn this into a progress notification.

Reply:
Oh also does this notification auto-dismiss? Not a big deal if it doesn't but something to think about. This is also why leaner UIs should be preferred (e.g. can the node just show that it's updating?)